### PR TITLE
Add Exercise07

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise07/Program.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        static void Main(string[] args)
+        {
+            var n = (Console.ReadLine().Any()) ? int.Parse(Console.ReadLine()) : 5;
+            var m = (Console.ReadLine().Any()) ? int.Parse(Console.ReadLine()) : 20;
+
+            var ints = Enumerable.Range(1, m-1);
+            var list = ints.OrderBy(x => Guid.NewGuid()).Take(n);
+
+            foreach (var i in list)
```
Comment (voquanghoa) [inline Exercise07/Program.cs]: Please change:
- Bad variable name. ==> rename
- Use `ForEach` function if posible

File: Exercise07/Program.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise07
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var n = (Console.ReadLine().Any()) ? int.Parse(Console.ReadLine()) : 5;
```
Comment (voquanghoa) [inline Exercise07/Program.cs]: Don't you see the bug here?

Comment (huynhngocminh2511) [reply Exercise07/Program.cs]: Done

File: Exercise07/Program.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise07/Program.cs]: Done

**Pull request merged**